apps/build-onchain-apps/src/utils/chainConfiguration.ts

```diff
@@ -0,0 +1,22 @@
+import { baseGoerli, Chain, goerli } from 'viem/chains';
+import { Environment } from './environment';
+import { getCurrentEnvironment } from './configuration';
+
+// The list of supported Chains for a given environment (e.g. Dev should only have testnet)
+export const supportedChains = new Map<Environment, Chain[]>([
+  [Environment.localhost, [baseGoerli]],
+  [Environment.development, [baseGoerli]],
+  [Environment.staging, [goerli]],
+  [Environment.production, [goerli]],
```

Comment:
nit. we want to be the default always being `base`, than others are optional choices.